common/BUILD.gn

```diff
@@ -48,28 +48,8 @@ source_set("switches") {
 }
 
 source_set("common") {
```

Comment:
I'm curious whether we need this source_set now? Can we move everything into `.gni`?

Reply:
Also (probably out of scope of this PR), but do we really need separate `source_set("switches")`, `source_set("network_constants")` etc in this file? @bridiver wdyt?

Reply:
well, it could be confusing to include `//chrome/common` for files in `//brave/common` so I think we need a `//brave/common` group with a public dep on `//chrome/common`. The whole point of this is to get rid of https://github.com/brave/brave-core/pull/6461/files#diff-5d387908947fadde91c9da6d1a66f66fR17 and that is necessary to avoid the circular dependency. It's just confusing the way it is now with some files still in this source_set, but most of them moved out

Reply:
I'm not sure if I understand this thread correctly. Do you guys mean that all sources_set files should be moved into sources.gni with only putting a common group into //brave/common/BUILD.gn?
```
group("common") {
  public_deps = [
    "//chrome/common",
  ]
}
```